pokeapp/src/components/Sidebar.js

```diff
@@ -25,8 +72,39 @@ const SideBar = ({
     return (
       <div className="sidebar-container">
         <div className="sidebar-dropdown">
-          <div className="dropdown"></div>
-          <div className="close-btn"></div>
+          <div
+            className="dropdown"
+            onClick={() => {
+              setShowPokemon(!showPokemon);
+            }}
+          >
+
+            <div
+              className="sidebar-dropdown-new"
+              onKeyPress={(e) => {
+                enterSubmit(e);
+              }}
+            >
+              <Typeahead
+                id="basic-typeahead-single"
+                labelKey="name"
+                options={pokeNameList}
+                placeholder={selectedPokemon.name}
```

Comment:
If there's no `selectedPokemon`, this may throw an error.  I'd set it to a static string like "Search..."